app/src/main/java/com/cornellappdev/android/pollo/NetworkUtils.java

```diff
@@ -31,9 +33,16 @@
     private static final String API_V2 = "/api/v2";
     private static final String GENERATE_CODE_ROUTE = "/generate/code";
     private static final String MOBILE_AUTH_ROUTE = "/auth/mobile";
+    private static final String JOIN_GROUP_ROUTE = "/join/session";
+    private static final String GET_GROUP_ROUTE = "/sessions/";
```

Comment:
```suggestion
    private static final String GET_GROUP_ROUTE = "/sessions";
```

There aren't trailing slashes anywhere else, it should be dropped here.

Reply:
gotcha, maybe there’s a way to name the variables so this is more clear? not too sure though